utils/test-cleanup.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+if [ "$(whoami)" != "root" ] ; then
+    sudo "$0"
+    exit 0
+fi
+
+echo 0 > /proc/sys/net/ipv4/ip_forward
```

Comment:
yeah... actually until tested on a gateway, anyway, the ones using the script will know what they do.